charts/kong/UPGRADE.md

```diff
@@ -41,7 +49,40 @@ text ending with `field is immutable`. This is typically due to a bug with the
 `init-migrations` job, which is [difficult to solve using current Helm
 functionality](https://github.com/Kong/charts/blob/master/charts/kong/FAQs.md#running-helm-upgrade-fails-because-of-old-init-migrations-job).
 If you encounter this error, deleting any existing `init-migrations` jobs will
-clear it.
+clear it. Set `migrations.init: false` in values.yaml to avoid this in future
+upgrades.
+
+## 1.5.0
+
+### Changes to migration job configuration
+
+Previously, all migration jobs were enabled/disabled through a single
+`runMigrations` setting. 1.5.0 splits these into toggles for each of the
+individual migrations:
+
+```
+migrations:
+  init: true
+  preUpgrade: true
+  postUpgrade: true
+  annotations:
+    sidecar.istio.io/inject: false
+    kuma.io/sidecar-injection: "disabled"
```

Comment:
Is it necessary to specify annotations here?
Most users don't need to worry about these annotations, and hence I'd recommend removing that part here.
Minimizing the amount of information that a user needs to digest during an upgrade is worthwhile.